server/app.js

```diff
@@ -28,93 +39,105 @@ app.use(bodyParser.urlencoded({ extended: false }));
 // parse application/json
 app.use(bodyParser.json());
 
+app.use(cors);
+
+app.use(intializeSesionStore());
+app.use(checkForExistingSessionAndAssignAccessKeys);
+
 /**
  * Create an SDK client for DynamoDB so we can use it to read/write records from the data store.
  */
-let config = {apiVersion: '2012-08-10'}
-// if we are running locally, add the required property for communicating with 
+let config = { apiVersion: "2012-08-10" };
```

Comment:
Should we have a codestyle file committed to the repo so we can have a standard way to format files?

Reply:
yup, good idea, we can add in the next commit